index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Algorithm Analysis Figures</title>
+    <style>
+        body {
+            height: 100%;
+            margin: 0;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            flex-direction: column;
+            gap: 2rem;
+        }
+        embed {
+            width: 70vw;
+            height: 100vh;
+        }
+    </style>
+</head>
+<body>
+    <h1>Algorithm Analysis Figures</h1>
+    <embed src="curve_plot.pdf" type="application/pdf" frameborder="0" scrolling="auto">
+    <embed src="D_limited.pdf" type="application/pdf" frameborder="0" scrolling="auto">
+    <embed src="D.pdf" type="application/pdf" frameborder="0" scrolling="auto">
+    <embed src="Dp_limited.pdf" type="application/pdf" frameborder="0" scrolling="auto">
+    <embed src="Dp.pdf" type="application/pdf" frameborder="0" scrolling="auto">
+    <embed src="durations.pdf" type="application/pdf" frameborder="0" scrolling="auto">
```

Comment:
Probably move curve and duration to the bottom as they're probably less interesting.